crmsh/parse.py

```diff
@@ -1742,4 +1745,13 @@ def parse(s, comments=None):
     except ParseError:
         return False
 
+
+def score_to_kind(score):
+    """
+    Convert score to kind for rsc_order
+    """
+    kind = "Optional"
+    if score == "INFINITY" or re.search("^[1-9][0-9]*$", score):
+        kind = "Mandatory"
```

Comment:
Pacemaker treats a score `0` as `Optional`,  and technically actually anything else as `Mandatory`.

Reply:
@liangxin1300 What do you think of the above comment?  Do you think it makes sense to make the logic here simpler and consistent with pacemaker:
https://github.com/ClusterLabs/pacemaker/blob/master/lib/pacemaker/pcmk_sched_constraints.c#L158-L166
?